_example/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 
 	config := server.Config{
 		Protocol: "tcp",
-		Address:  "localhost:3306",
+		Address:  "localhost:3307",
```

Comment:
Why the change to 3307?

Reply:
Oops this was just for it to work locally (MySQL occupies 3306). I'll change it back.